tests/integration/v3_watch_test.go

```diff
@@ -968,6 +968,8 @@ func TestV3WatchProgressOnMemberRestart(t *testing.T) {
 
 	t.Log("Waiting for result")
 	select {
+	case <-progressNotifyC:
```

Comment:
the `lastWatchedModRevision` > 0 before restart member. It seems it's unlikely to send the item into `progressNotifyC`

Reply:
Added the sleep after restart and it can reproduce it.

```diff
--- a/tests/integration/v3_watch_test.go
+++ b/tests/integration/v3_watch_test.go
@@ -961,6 +961,7 @@ func TestV3WatchProgressOnMemberRestart(t *testing.T) {
        clus.Members[0].Restart(t)
        clus.Members[0].WaitOK(t)

+       time.Sleep(2 * time.Second)
        // write the same key again after the member restarted
        t.Log("Writing the same key 'foo' again after restarting the member")
        _, err = client.Put(ctx, "foo", "bar2")
```

Reply:
yes, as long as the goroutine doesn't get blocked on sending progress notification, we are good.

Reply:
let's keep the fix as simple as possible.